kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ProxyDeploymentDependentResource.java

```diff
@@ -248,4 +257,12 @@ static String getOperandImage() {
         }
 
     }
+
+    @NonNull
+    private static Map<String, Quantity> defaultLimits() {
```

Comment:
Honestly, it seems a bit odd to declare a method _and_ have constants. One or the other would be sufficient.

Reply:
I had a misconception that `Map.of` was a hashmap, it actually looks like it will preserve insertion order (trying to keep the output deterministic so it's 1:1 with our DerivedResourceTests expectation yaml). Reverting to that.

Reply:
The javadoc says the order of `Map.of` is "unspecified and subject to change". So I'd stick with `LinkedHashSet`, but use a `static` block to init the constant:

```java
static {
 ...
}
```